diff --git a/node_modules/react-native-circular-action-menu/ActionButton.js b/node_modules/react-native-circular-action-menu/ActionButton.js
index eb64e59..ae4cdd4 100644
--- a/node_modules/react-native-circular-action-menu/ActionButton.js
+++ b/node_modules/react-native-circular-action-menu/ActionButton.js
@@ -60,6 +60,7 @@ export default class ActionButton extends Component {
     return [styles.overlay, styles.actionContainer, {
       alignItems,
       justifyContent,
+	  padding: this.props.size/2,
     }];
   }
   getActionsStyle() {
@@ -306,7 +307,6 @@ const styles = StyleSheet.create({
   },
   actionContainer: {
     flexDirection: 'column',
-    padding: 10,
   },
   actionBarItem: {
     alignItems: 'center',
